src/components/Room.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled, { createGlobalStyle } from "styled-components";
+
+export const StyledMain = styled.main`
+	background: #c7ddc9;
+	height: 93vh;
+	padding: 1%;
+`;
+
+export const GlobalStyle = createGlobalStyle`
```

Comment:
Is there a reason you used a `createGlobalStyle` instead of wrapping `<App/>` in `main.tsx` in a Styled component that removes margins?

I think all the pages have the margin issue from what I see so all of them have to be reset to `margin: 0`. 

If there is any other usecase for `createGlobalStyle`, let me know.

Reply:
Is there a use case for using `createGlobalStyle` over just a div that has this style? Something like `StyledAppWrapper` that wraps the `<App/>` component in main.tsx would seem simpler. 

I cant seem to find documentation on why you would use `createGlobalStyle` over a wrapper div, but it does seem like it would be used for theming, which we arent doing......yet.

Reply:
Well when I searched on how to adjust the global body, I found out global styles. 
https://scalablecss.com/styled-components-global-styles/ 

I'm trying to do Styled app Wrapper but it just adds another div which is not targeting the main body of the whole app 
it's not working
![image](https://github.com/CommunityFocus/cf-frontend/assets/82416131/b9414dba-2cbb-413b-848e-63fe40eb40dd)
Is that right coding wise?
![image](https://github.com/CommunityFocus/cf-frontend/assets/82416131/7b199205-7b53-489a-8a6a-d1222612fb18)
![image](https://github.com/CommunityFocus/cf-frontend/assets/82416131/3fc70b85-1e55-4716-a37d-ca7f31288567)


The body still has margins
![image](https://github.com/CommunityFocus/cf-frontend/assets/82416131/c448a946-2625-4abd-899b-0be9de4576c8)

Reply:
Thank you for linking the resource! This seems to be the correct way of doing it.

Reply:
So What do you mean? 

the styled app Wrapper adds another div, which is incorrect, I think. it's not targeting the main body to remove the margins, what to do?

Reply:
Sorry for the late reply. I read up more and tried it with the wrapper. You are right, there doesnt seem to be a way to target the body unless you use `createGlobalStyle`.  @AhmedHHamdy 

I approved and merged the PR.